### DISCUSSION TITLE
Diagnostics:2 failed plus 0 muted

### DISCUSSION THREAD

**Discussion by jack-liew** [rancher-sandbox/rancher-desktop#3648] [Q&A]
![image](https://user-images.githubusercontent.com/102201538/208148633-4bb9cdca-6a6f-4d3e-b922-1b50920854b3.png)

What shall I do? RD for Windows.
Help~~~

**Comment by mook-as** [comment]
We haven't finished building on the features to link to suggestions yet (it's planned, but not done).  In the mean time:
- Please ensure you can access the internet. (It's trying to talk to `https://example.com/`, if that helps.)
- Please ensure WSL is installed correctly: please run `wsl --update` in an elevated command prompt, and handle any issues that come up. We may have bugs around non-English versions of Windows, though (we may not be detecting the kernel version correctly if the output is localized); in that case, please just mute the diagnostic.

**Reply by jack-liew** [reply]
I install the latest RD 1.7.0.
My WSL everything is normal, it can access to internet address, like www.bing.com.
and my RD works fine.

but, RD‘Diagnostics TAB always displays both errors.

When i reinstall RD v1.62 and start it, the error tip:
`capture output: 
wsl.exe -d /bin/sh -c [ -e os-ver..........]`
When I run this command on the command line, Must be rewritten as:
`wsl.exe -d /bin/sh -c "[ -e os-ver..........]"`

Thank U
